app/src/main/res/values/strings.xml

```diff
@@ -34,12 +34,107 @@
 
     <string name="address">Address</string>
     <string name="city">City</string>
-    <string name="area">Postal code</string>
+    <string name="area">Area</string>
     <string name="country">Country</string>
     <string name="first_name">First name</string>
     <string name="last_name">Last name</string>
     <string name="billing_address">Billing address</string>
     <string name="required_field">Required field</string>
+    <string name="get_relevant_information">Get relevant information</string>
+    <string name="weather_forecast">Weather forecast</string>
+    <string name="roads_amp_traffic_info"><![CDATA[Roads & traffic info]]></string>
+    <string name="title_activity_fuel_prices">FuelPricesActivity</string>
+    <string name="large_text">
+        "Material is the metaphor.\n\n"
+
```

Comment:
Zašto je ovako puno teksta unutar String file-a?

Reply:
To je automatski generiran tekst kad želiš vidjeti kako u app izgleda veliki komad teksta.